src/subhalo_unbind.cpp

```diff
@@ -281,6 +281,23 @@ inline void RefineBindingEnergyOrder(EnergySnapshot_t &ESnap, HBTInt Size, Gravi
 }
 void Subhalo_t::Unbind(const Snapshot_t &epoch)
 { // the reference frame (pos and vel) should already be initialized before unbinding.
+
+  /* We skip already existing orphans */
+  if(!Particles.size())
```

Comment:
Should I change this condition to IsAlive()? That would make it consistent with what I say in a comment (i.e. can we have a Particles.size() == 0 if the subgroup has not disrupted in the past? I do not think so, based on the re-adding of tracer particles during masking